doc/USING.rst

```diff
@@ -28,12 +28,12 @@ The GPU capable builds (Python, NodeJS, C++, etc) depend on the same CUDA runtim
 Getting the pre-trained model
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If you want to use the pre-trained English model for performing speech-to-text, you can download it (along with other important inference material) from the DeepSpeech `releases page <https://github.com/mozilla/DeepSpeech/releases>`_. Alternatively, you can run the following command to download and unzip the model files in your current directory:
+If you want to use the pre-trained English model for performing speech-to-text, you can download it (along with other important inference material) from the DeepSpeech `releases page <https://github.com/mozilla/DeepSpeech/releases>`_. Alternatively, you can run the following command to download the model files in your current directory:
 
 .. code-block:: bash
 
-   wget https://github.com/mozilla/DeepSpeech/releases/download/v0.6.1/deepspeech-0.6.1-models.tar.gz
-   tar xvfz deepspeech-0.6.1-models.tar.gz
+   wget https://github.com/mozilla/DeepSpeech/releases/download/v0.7.0/deepspeech-0.7.0-models.pbmm
```

Comment:
Same, TFLite ?

Reply:
There are no other references to TFLite in this file and I tried to keep changes to a minimum here given that it's the PR bumping the version to a new release. Can we do that in the 0.8 cycle?